drivers/spi/spi_nxp_lpspi/spi_nxp_lpspi_common.c

```diff
@@ -11,6 +11,8 @@
  * the driver is going to achieve the zephyr API.
  */
 
+#define DT_DRV_COMPAT nxp_lpspi
```

Comment:
Please add a comment why this is here, so nobody in the future thinks it's save to remove.